src/App.tsx

```diff
@@ -44,15 +49,59 @@ export default function App() {
             width: 30em;
           `}
         >
-          <Upload
-            uploadedPhoto={uploadedPhoto}
-            setUploadedPhoto={(photo) => {
-              setPreset(null);
-              setUploadedPhoto(photo);
-            }}
-          />
+          <Flexbox gap={12} direction="column">
```

Comment:
can you reuse the top level flexbox? unsure about this whole layout tbh so no worries